docs/unified-test-documentation/dasharo-stability/capsule-update.md

```diff
@@ -0,0 +1,337 @@
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Obtain and save the BIOS version as `${original_bios_version}`.
+1. Perform a capsule update using an invalid GUID file.
+1. Reboot the system and select the UEFI Shell boot option.
+1. After the system boots, verify and save the BIOS version as
+   `${updated_bios_version}`.
+1. Check that the BIOS version has not changed by verifying that
+   `${original_bios_version}` is equal to `${updated_bios_version}`.
+1. Verify that the status contains a `Not Ready` indication.
+
+**Expected result**
+
+1. The BIOS version should not change after attempting the capsule update.
+1. The status should contain a note indicating that the update was rejected due
+   to the invalid GUID.
+
+---
+
+## CUP130.001 Verifying BIOS Settings Persistence After Update - PART 1
+
+**Test description**
+
+This test aims to verify that BIOS settings persist after a Capsule Update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Enter the Setup Menu.
+1. Navigate to Boot Maintenance Manager.
+1. Change the Auto Boot Time-out value to an arbitrary custom one, such as
+   32123
+1. Save the settings and reboot the system.
+
+**Expected result**
+
+1. The custom Auto Boot Time-out value should remain set after the reboot.
+
+---
+
+## CUP150.001 Capsule Update
+
+**Test description**
+
+This test verifies a successful capsule update and ensures the correct user
+experience (UX) during the update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. Custom logo firmware image
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Obtain and save the BIOS version as `${original_bios_version}`.
+1. Perform a capsule update using a valid capsule.
+1. Verify that the update screen looks as expected.
+1. Reboot the system and select the UEFI Shell boot option.
+1. After the system boots, verify the BIOS version and save it as
```

Comment:
This happens automatically as part of the update, and this is done by firmware and not operating system, so:

```suggestion
1. After the platform reboots, select the UEFI Shell boot option.
1. Verify the BIOS version and save it as
```

Reply:
[Resolved](https://github.com/Dasharo/docs/pull/915/commits/ab710300656f097092f77f396565534fc09639a4)

Reply:
Almost, `verify` -> `Verify`.

Reply:
Fixed in https://github.com/Dasharo/docs/compare/ab710300656f097092f77f396565534fc09639a4..5eb4891dff7a1984e86cf5049fd1d00d5c9aa0a7